--- a/src/caelestia/utils/scheme.py
+++ b/src/caelestia/utils/scheme.py
@@ -173,6 +173,8 @@
             self._colours = read_colours_from_file(self.get_colours_path())
 
     def __str__(self) -> str:
+        _sep = '\n        '
+        _esc = '\x1b'
         return (
             f"Current scheme:\n"
             f"    Name: {self.name}\n"
@@ -180,7 +182,7 @@
             f"    Mode: {self.mode}\n"
             f"    Variant: {self.variant}\n"
             f"    Colours:\n"
-            f"        {'\n        '.join(f'{n}: \x1b[38;2;{int(c[0:2], 16)};{int(c[2:4], 16)};{int(c[4:6], 16)}m{c}\x1b[0m' for n, c in self.colours.items())}"
+            f"        {_sep.join(f'{n}: {_esc}[38;2;{int(c[0:2], 16)};{int(c[2:4], 16)};{int(c[4:6], 16)}m{c}{_esc}[0m' for n, c in self.colours.items())}"
         )
 
 
